modules/gapi/src/backends/fluid/gfluidcore_func.hpp

```diff
@@ -129,6 +129,17 @@ MULC_SIMD(float, float)
 
 #undef MULC_SIMD
 
+#define ABSDIFFC_SIMD(T)                                            \
+int absdiffc_simd(const T in[], const float scalar[], T out[],      \
```

Comment:
what are differences with function in dispatch https://github.com/opencv/opencv/pull/21204/files#diff-693224a96b2408435e643bd3353f4a66954b6d57accd21fac061d14677d5c656R169 ?

Reply:
There are no difference. This structure was organized for dynamic dispatching support.

Reply:
Could you describe relationship by short schema between them please? 
Because i see two functions with the similar names, with the similar agruments and in the same namespace .
May be one of fucntions should have *_impl suffic, may be i wrong

Reply:
Am i right in my assumption (it's definitely clear for me by now) that here we have function declaration
but in file before we have function definition?

If yes, then i suggest to rename MACRO as `ABSDIFFC_SIMD_DECLARATION` & `ABSDIFFC_SIMD_DEFINITION` or something like that

Reply:
`gfluidcore_func.hpp` is header file for forward declaration of wrappers in the `gfluidcore_func.dispatch.cpp`. It's necessary for C++ compiler.

Reply:
> i suggest to rename MACRO as `ABSDIFFC_SIMD_DECLARATION` & `ABSDIFFC_SIMD_DEFINITION` or something like that

Ok. I would prefer do this in the next PR together with MACRO for other kernels SIMD. All together.